index.html

```diff
@@ -0,0 +1,131 @@
+        <i class="fa-solid fa-xmark closebtn close-menu mobile"></i>
+        <div class="overlay-content">
+          <a href="about.html" class="close-menu">About</a>
+          <a href="#top-songs" class="close-menu">Top Songs</a>
+          <a href="#featured-performers" class="close-menu">Performers</a>
+          <a href="#partners" class="close-menu">Partners</a>
+        </div>
+      </div>
+      </div>
+    </header>
+
+    <section id="top-songs" class="top-songs">
+      <h2>Top songs worldwide</h2>
+      <div class="musics-container">
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/harry-styles-bma-as-it-was-po3-344x344.jpg" alt="">
+            <span class="music-author">As It Was</span>
+            <p class="music-description">Harry Styles</p>
+          </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/lizzo-o9x-about-damn-time-f6q-344x344.jpg" alt="">
+            <span class="music-author">Lizzo </span>
+            <p class="music-description">About Damn Time </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/jack-harlow-i43-344x344.jpg" alt="">
+            <span class="music-author">Jack Harlow </span>
+            <p class="music-description">First Class </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/kate-bush-qv2-running-up-that-hill-1cx-344x344.jpg" alt="">
+            <span class="music-author">Kate Bush </span>
+            <p class="music-description">Running Up That Hill (A Deal With God)</p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/future-f8b-344x344.jpg" alt="">
+            <span class="music-author">Future Featuring Drake & Tems  </span>
+            <p class="music-description">Wait For U </p>
+        </div>
+        </a>
+        <a href="#">
+          <div class="music">
+            <img class="music-art" src="./images/drake-04g-344x344.jpg" alt="">
+            <span class="music-author">Drake Featuring 21 Savage </span>
+            <p class="music-description">Jimmy cooks</p>
+          </div>
+        </a>
+      </div>
+    </section>
+
+    <section id="featured-performers" class="featured-performers">
+      
+    </section>
+
+    <section id="partners" class="partners">
+      <h2>Partners</h2>
+      <div class="partners-container">
+        <img src="./images/pepsi-icon.svg" alt="pepsi-logo">
+        <img src="./images/espn-logo.svg" alt="espn-logo">
+        <img src="./images/puma-logo.svg" alt="puma-logo">
+        <img src="./images/mtv-live-seeklogo.com.svg" alt="mtv-logo">
+        <img src="./images/redbullenergydrink.svg" alt="red-bull">
+      </div>
+    </section>
```

Comment:
- [ ] `required`
- [ ]  just as mentioned on top I suggest you follow the original template here as well. please see screenshots below

`figma template`
![speakers](https://user-images.githubusercontent.com/66230306/181393830-076c378c-7b0b-47a2-a67e-f1c116cd0198.png)

`your template`

![ft speakers](https://user-images.githubusercontent.com/66230306/181394001-e79495e8-1915-4aca-b747-9bee90da44e5.png)